core/templates/pages/skill-editor-page/skill-editor-page-root.component.html

```diff
@@ -0,0 +1,20 @@
+<oppia-angular-root>
+  <oppia-base-content>
+    <navbar-breadcrumb>
+      <oppia-skill-editor-navbar-breadcrumb>
+      </oppia-skill-editor-navbar-breadcrumb>
+    </navbar-breadcrumb>
+
+    <nav-options>
+      <oppia-skill-editor-navbar>
+      </oppia-skill-editor-navbar>
+    </nav-options>
+
+    <content>
+      <oppia-skill-editor-page></oppia-skill-editor-page>
+    </content>
+
+    <page-footer></page-footer>
+  </oppia-base-content>
+</oppia-angular-root>
+<script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=default"></script>
```

Comment:
Are you able to use MathJax on skill-editor page? (Open any RTE and try adding math formula)

Reply:
Yes. (As discussed in our previous meeting)